internal/apiserver/route_get_verifier_by_id.go

```diff
@@ -0,0 +1,44 @@
+// Copyright © 2022 Kaleido, Inc.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package apiserver
+
+import (
+	"net/http"
+
+	"github.com/hyperledger/firefly/internal/config"
+	"github.com/hyperledger/firefly/internal/i18n"
+	"github.com/hyperledger/firefly/internal/oapispec"
+	"github.com/hyperledger/firefly/pkg/fftypes"
+)
+
+var getVerifierByID = &oapispec.Route{
+	Name:   "getVerifierByID",
+	Path:   "namespaces/{ns}/verifiers/{vid}",
+	Method: http.MethodGet,
+	PathParams: []*oapispec.PathParam{
+		{Name: "ns", ExampleFromConf: config.NamespacesDefault, Description: i18n.MsgTBD},
+		{Name: "vid", Example: "id", Description: i18n.MsgTBD},
```

Comment:
Just want to note that this became a "hash" rather than an "id" in the latest pass. Might suggest a tweak to reflect that on the Swagger?

Reply:
I actually thought about that and didn't do it. My thinking was that it's actually a resource identifier, just one that's deterministically generated on all nodes. We don't rely on it being a hash for processing, and due to an edge case on migration there's a small number of environments where it might not actually be the Hash.

Reply:
OK - I'm mainly worried about the case where I query `/verifiers` and get a list of things, how do I know which one to put in `vid` in order to look up that one item? I have to know that `hash` maps to `vid`. Of course this is a detail that can easily be tweaked later.